pkg/apis/proxy/v1alpha1/managedproxyconfiguration_types.go

```diff
@@ -189,25 +203,28 @@ type ManagedProxyConfigurationProxyServer struct {
 	// Note that the replicas will also be reflected in the flag `--server-count`
 	// so that agents can discover all the server instances.
 	// +optional
-	// +kubebuilder:default=3
```

Comment:
@qiujian16 i missed this line, is it intended?

Reply:
probably not...